tests/unit/lib/test_opensearch_tls.py

```diff
@@ -89,6 +89,9 @@ def setUp(self, _) -> None:
         self.harness.add_network("1.1.1.1", endpoint=TLS_RELATION)
         self.harness.begin()
         self.charm = self.harness.charm
+        self.charm.tls._remove_ca_from_request_bundle = MagicMock()
+        self.charm.tls._add_ca_to_request_bundle = MagicMock()
```

Comment:
@skourta I cannot get the TLS tests passings without mocking those two methods. Is it a problem to mock these values?